schema.md

```diff
@@ -0,0 +1,555 @@
+width.
+
+### Key `description/shortdesc`
+
+* Type: multi-language string (max 100 chars)
+* Presence: mandatory
+* Example: "Advanced booking system for hospitals"
+
+This key contains a short description of the software. It should be
+a single line containing a single sentence. Maximum 100 characters are
+allowed.
+
+### Key `description/longdesc`
+
+* Type: multi-language string (min 500 chars, max 10000 chars)
+* Presence: mandatory
+
+This key contains a longer description of the software, between 500
+and 10000 chars. It is meant to provide an overview of the capabilities
+of the software for a potential users. The audience for this text
+should be that of users of the software, not developers. You can think
+of this text as the description of the software that would be in its
+website (if the software had one).
+
+### Key `description/screenshots`
+
+* Type: array of strings (paths)
+* Presence: optional
+* Formats: PNG, JPG
+* Example: "data/screenshots/configuration.png"
+
+This key contains one or multiple paths to files showing screenshots of
+the software. They are meant to give a quick idea on how the software
+looks like and how it works.
+
+Screenshots can be of any shape and size; the suggested formats are:
+
+ * Desktop: 1280x800 @1x
+ * Tablet: 1024x768 @2x
+ * Mobile: 375x667 @2x
+
+
+### Key `description/videos`
+
+* Type: array of strings (URLs)
+* Presence: optional
+* Example: "https://youtube.com/xxxxxxxx"
+
+This key contains one or multiple URLs of videos showing how the
+software works. Like screenshots, videos should be used to give a quick
+overview on how the software looks like and how it works. Videos should
+be hosted on a video sharing website that supports the
+[oEmbed](https://oembed.com) standard; popular options are YouTube and
+Vimeo.
+
+
+
+
+
+
+
+
+
+
+
+## Section `meta`
+
+This section contains information useful to classify the software,
+define its scope of usage and its goal at a general level.
+
+### Key `meta/scope`
+
+* Type: string or array of strings
+* Presence: optional
+* Example: "es"
+
+This key defines the expected geographical scope of the software. If
+a software is probably useful only within some specific countries
+(because, eg., it helps implementing a specific regulation), specify
+the ISO693-1 two-letter country codes of the countries n this key.
+
+### Key `meta/pa-type`
+
+* Type: enumerated string or array of strings
+* Presence: optional
+* Values: see [pa-types.md](pa-types.md)
+* Example: "city"
+
+This key defines the types of public administration which is expected
+to use this software. Public software is often specific in scope, because
+there is a large set of tasks that are specific to each type of
+administration. For instance, many softwares that are used in schools
+are probably not useful in hospitals.
+
+The list of allowed values is defined in [pa-types.md](pa-types.md),
+and can evolve at any time, separately from the version of this
+specification.
```

Comment:
This is a very complex one, and I'm looking forward to working on figuring this out a bit more, since even words like `school` or `city` vary wildly in meaning between contexts. But this is good for now!

Reply:
Sure we can evolve from here.